src/main/java/mate/jdbc/dao/impl/CarDaoImp.java

```diff
@@ -0,0 +1,182 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImp implements CarDao {
+    @Override
```

Comment:
you need to add getAllByDriver to this class

Reply:
I did not find strict recommendations on this issue, so I want to refer to the methods and clarify, while I will leave it as it is)